android/Dockerfile

```diff
@@ -0,0 +1,124 @@
+FROM golang:1.14.4
+
+# CockroachDB
+
+RUN wget -qO- https://binaries.cockroachdb.com/cockroach-v20.1.1.linux-amd64.tgz | tar  xvz
+RUN cp -i cockroach-v20.1.1.linux-amd64/cockroach /usr/local/bin/
```

Comment:
I think 20.1.3 is available :)

Reply:
this part can be removed for now. Android integration tests are using only postgres currently.